.gitignore

```diff
@@ -130,3 +130,10 @@ dmypy.json
 
 # Vagrant
 .vagrant/
+<<<<<<< HEAD
```

Comment:
This isn't code.
This is git way to tell you that you have a conflict that needs to be resolved

Reply:
hi, we've already fixed the conflicts. is there any new ones?

Reply:
The '<<<< HEAD' and '>>>>>' were created by git to show you the area that needs to be resolved,
When you resolve it, you need to remove these lines.
Usually (unless it's a text file), the code won't work with these lines inside.